activejob/lib/active_job/enqueuing.rb

```diff
@@ -10,8 +10,8 @@ module Enqueuing
     # Includes the +perform_later+ method for job initialization.
     module ClassMethods
       # Push a job onto the queue. The arguments must be legal JSON types
-      # (+string+, +int+, +float+, +nil+, +true+, +false+, +hash+ or +array+) or
-      # GlobalID::Identification instances. Arbitrary Ruby objects
+      # (+string+, +int+, +float+, +nil+, +true+, +false+, +hash+ or +array+),
```

Comment:
Actually the legal JSON types is not perfectly true, JSON doesn't have `int` and `float`, but have `number`.
Probably the original intent for the doc is that the legal JSON types == PERMITTED_TYPES.

Reply:
Yes.... I wasn't also sure about that, let me better remove it.